addons/mrp/models/mrp_production.py

```diff
@@ -275,6 +275,15 @@ def create(self, values):
         return production
 
     @api.multi
+    def write(self, vals):
```

Comment:
I meant: if something changes on the lines of the move_raw_ids/move_finished_ids of the production order, it should be logged on the production order.  Maybe what you did here, can be done (and by track_visibility) if not already the case.

Reply:
Point 1 : For production order we can use track_visibility
Point 2:  We are not able to change move_raw_ids/move_finished_ids  when mo is done so we dont 
need to put message on move lines changes ( move only change when we update production quantity).

So I think we dont need to put message for (move_raw_ids/move_finished_ids) production order.

Reply:
Point 2 -> When my branch will be merged, you will be able to change it.